include/zephyr/bluetooth/audio/tbs.h

```diff
@@ -326,10 +326,10 @@ int bt_tbs_set_uri_scheme_list(uint8_t bearer_index, const char **uri_list,
  */
 void bt_tbs_register_cb(struct bt_tbs_cb *cbs);
 
-#if defined(CONFIG_BT_DEBUG_TBS)
+#if defined(CONFIG_BT_TBS_LOG_LEVEL_DBG)
 /** @brief Prints all calls of all services to the debug log */
 void bt_tbs_dbg_print_calls(void);
```

Comment:
I would check if you need those ifdefs at all. When debug logs are off function is empty and never called. I would expect that linker handles it.

Reply:
Fixing this is not part of this PR.